src/main/java/org/gridsuite/voltageinit/server/computation/service/AbstractComputationObserver.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.voltageinit.server.computation.service;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.NonNull;
+
+/**
+ * @author Mathieu Deharbe <mathieu.deharbe at rte-france.com>
+ * @param <R> powsybl Result class specific to the computation
+ * @param <P> powsybl and gridsuite parameters specifics to the computation
+ */
+public abstract class AbstractComputationObserver<R, P> {
+    protected static final String OBSERVATION_PREFIX = "app.computation.";
+    protected static final String PROVIDER_TAG_NAME = "provider";
+    protected static final String TYPE_TAG_NAME = "type";
+    protected static final String STATUS_TAG_NAME = "status";
+    protected static final String COMPUTATION_COUNTER_NAME = OBSERVATION_PREFIX + "count";
+    protected final ObservationRegistry observationRegistry;
+    protected final MeterRegistry meterRegistry;
```

Comment:
```suggestion
    @Getter private final ObservationRegistry observationRegistry;
    @Getter private final MeterRegistry meterRegistry;
```
(move `@Getter` on class)